<launch>

  <node pkg="bender_arm_control" type="arm_teleop.py" name="simple_arm_teleop" output="screen">
    <rosparam param="arm_name">["l_arm"]</rosparam>
    <param name="rate" type="int" value="5" />
  </node>

  </launch>
  